packages/playground/windows/playground-composition/Playground-Composition.cpp

```diff
@@ -273,13 +166,83 @@ struct WindowData {
 
           winrt::Microsoft::ReactNative::ReactViewOptions viewOptions;
           viewOptions.ComponentName(appName);
-          m_CompositionHwndHost.ReactViewHost(
+          auto windowData = WindowData::GetFromWindow(hwnd);
+
+          if (!m_compRootView) {
+            if (windowData->m_useLiftedComposition) {
+              m_compRootView = winrt::Microsoft::ReactNative::CompositionRootView(g_liftedCompositor);
+            } else {
+              m_compRootView = winrt::Microsoft::ReactNative::CompositionRootView();
+            }
+          }
+
+          m_compRootView.ReactViewHost(
               winrt::Microsoft::ReactNative::ReactCoreInjection::MakeViewHost(host, viewOptions));
 
-          auto windowData = WindowData::GetFromWindow(hwnd);
-          if (!windowData->m_windowInited) {
-            m_CompositionHwndHost.Initialize((uint64_t)hwnd);
-            windowData->m_windowInited = true;
+          if (windowData->m_useLiftedComposition) {
+            // By using the MicrosoftCompositionContextHelper here, React Native Windows will use Lifted Visuals for its
+            // tree.
+            winrt::Microsoft::ReactNative::Composition::CompositionUIService::SetCompositionContext(
+                InstanceSettings().Properties(),
+                winrt::Microsoft::ReactNative::Composition::MicrosoftCompositionContextHelper::CreateContext(
+                    g_liftedCompositor));
+
+            auto bridge = winrt::Microsoft::UI::Content::DesktopChildSiteBridge::Create(
+                g_liftedCompositor, winrt::Microsoft::UI::GetWindowIdFromWindow(hwnd));
+
+            auto appContent = m_compRootView.Island();
+
+            auto invScale = 1.0f / ScaleFactor(hwnd);
+            m_compRootView.RootVisual().Scale({invScale, invScale, invScale});
+
+            /*
+              // Future versions of WinAppSDK will have more capabilities around scale and size
+              auto site = bridge.Site();
+              auto siteWindow = site.Environment();
+              auto displayScale = siteWindow.DisplayScale();
+
+              site.ParentScale(displayScale);
+              site.ActualSize({metrics.Frame.Width / 2, metrics.Frame.Height / 2});
+              site.ClientSize(
+                  {static_cast<int32_t>(metrics.Frame.Width / 2 * metrics.PointScaleFactor),
+                  static_cast<int32_t>(metrics.Frame.Height / 2 * metrics.PointScaleFactor)});
+            */
```

Comment:
Is there a tracking item in WinUI's repo/roadmap that we can add a link to here?